internal/pkg/cli/env_deploy.go

```diff
@@ -256,6 +258,7 @@ After fixing the deployment, you can:
 		}
 		return fmt.Errorf("deploy environment %s: %w", o.name, err)
 	}
+	log.Ssuccessf("Deployed environment %s.\n", color.HighlightUserInput(o.name))
```

Comment:
we probably need detached flag checking there to exit earlier.

Reply:
I think we do not think that because we have an earlier return stmt here 
https://github.com/aws/copilot-cli/pull/5123/files#diff-ed267edf51bd253fe120a5d1b9ed09b357b1c61eafac3501a95642aced180ee1R237
and here
https://github.com/aws/copilot-cli/pull/5123/files#diff-ed267edf51bd253fe120a5d1b9ed09b357b1c61eafac3501a95642aced180ee1R241

Reply:
I just moved the log statement 
```
if err == nil {
  log.Successf("succesfully deployed Environment")
  return nil
}